test_runner/src/main/kotlin/ftl/json/OutcomeDetailsFormatter.kt

```diff
@@ -14,9 +14,11 @@ import ftl.util.StepOutcome.success
 import ftl.util.StepOutcome.unset
 
 internal fun Outcome?.getDetails(
-    testSuiteOverviewData: TestSuiteOverviewData?
+    testSuiteOverviewData: TestSuiteOverviewData?,
+    isRoboTest: Boolean = false
 ): String = when (this?.summary) {
-    success, flaky -> testSuiteOverviewData
+    success, flaky -> if (isRoboTest) "---"
+    else testSuiteOverviewData
         ?.getSuccessOutcomeDetails(successDetail?.otherNativeCrash ?: false)
         ?: "Unknown outcome"
```

Comment:
could you move 
```kotlin
testSuiteOverviewData
        ?.getSuccessOutcomeDetails(successDetail?.otherNativeCrash ?: false)
        ?: "Unknown outcome"
```
to separate method and make single line
```kotlin
success, flaky -> if (isRoboTest) "---" else newMethod()
```

for better readability ?

Reply:
Like your proposal, thanks for catching this!